PLAT-11769 adding welcome template styles inside the code file (#914)
crowdbotics/modules: scaffold/CHANGELOG.md

Based on [Common Changelog](https://common-changelog.org/).

## 2.3.0 - 2023-07-10

### Changed

> - update the welcome screen template, style at thecode file

Reviewer: ```suggestion
- update the welcome screen, move styles to the `index.js` file
```
Suggestion
